src/Directory.Packages.props

```diff
@@ -71,7 +77,10 @@
     -->
     <GlobalPackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" />
     <GlobalPackageReference Include="PolySharp" Version="1.8.1" />
-    <GlobalPackageReference Include="TunnelVisionLabs.ReferenceAssemblyAnnotator" Version="1.0.0-alpha.160" />
-    <PackageDownload Include="Microsoft.NETCore.App.Ref" Version="[5.0.0]" />
```

Comment:
If the rest of the PR is unwanted, at a minimum suggest this version should bump to 6.0.14